backend/src/Edna/Web/Handlers.hs

```diff
@@ -0,0 +1,99 @@
+module Edna.Web.Handlers
```

Comment:
I think this module should have almost no logic and just define `ednaHandlers` with simple definitions. `eeUploadExperiment = bla-bla` where `bla-bla` is something small and uses logic from another module. So in this case I think `parseExperimentXls` should be in a different module, somewhere outside of `Edna.Web`. I think @pva701 just didn't care much about the structure of modules here, which is ok for this PR (as you said).

Overall I agree with this 3-layer approach, but currently there is just no code to apply it :)

Reply:
good point, though, I can't separate this uploading logic from the rest of (future) logic (perhaps because there is no logic yet and I can't come up with it). so I more or less agree but don't want to do it in this PR :putya:

Reply:
What about just creating a new module and putting `parseExperimentXls` there?

Reply:
Whatever, we'll do it along with implementing proper parser (now that we have example of inputs).